plugin/adsk/scripts/AEmayaUsdProxyShapeBaseTemplate.mel

```diff
@@ -26,6 +26,36 @@ global proc AEmayaUsdProxyShapeInfoReplace(string $input)
     textFieldGrp -edit -text $defaultPrim mayaUsdProxyShapeDefaultPrim;
 }
 
+global proc AEmayaUsdProxyShapePayloadsNew(string $loadPayloads)
+{
+    setUITemplate -pst attributeEditorTemplate;
+    rowLayout -nc 5 -cl5 "right" "center" "center" "center" "center";
+        text -label `getMayaUsdString("kPayloads")` -ann `getMayaUsdString("kPayloadsAnn")`;
+        button -label `getMayaUsdString("kLoadAll")` payloadsLoadAllBtn;
+        button -label `getMayaUsdString("kUnloadAll")` payloadsUnloadAllBtn;
+        text -label "";
+        text -label "";
+    setParent ..;
+    setUITemplate -ppt;
+    AEmayaUsdProxyShapePayloadsReplace($loadPayloads);
+}
+
+global proc AEMayaUsdProxyShapePayloadsCmd(string $fullNodeName, int $loadAll)
+{
+    python("import AETemplateHelpers; AETemplateHelpers.ProxyShapePayloads('" + $fullNodeName + "', " + $loadAll + ")");
+}
```

Comment:
Wrap the call to python into a MEL proc. This way its the MEL proc name that will appear in the undo/redo menu (rather than the python line).